_examples/01_minimum/federation/federationservice_grpc_federation.go

```diff
@@ -207,6 +349,30 @@ func recoverErrorFederationService(v interface{}, rawStack []byte) *FederationSe
 	}
 }
 
+func (s *FederationService) evalCEL(expr string, vars []cel.EnvOption, args map[string]any, outType reflect.Type) (any, error) {
+	env, err := s.env.Extend(vars...)
+	if err != nil {
+		return nil, err
+	}
+	expr = strings.Replace(expr, "$", "__ARG__", -1)
```

Comment:
nit, `__ARG__ ` appears a few times in the code, so I believe it would be better if we constantize it and explain what it's for

Reply:
nit, might be better if we check the expr contains `__ARG__` before replacing `$` with it, even though that is quite unlikely users use it, but just in case...